tests/foreman/cli/test_host.py

```diff
@@ -161,17 +162,15 @@ def test_positive_create_with_name(self):
             with self.subTest(name):
                 host = entities.Host()
                 host.create_missing()
-                result = Host.create({
+                result = make_host({
                     u'architecture-id': host.architecture.id,
                     u'domain-id': host.domain.id,
                     u'environment-id': host.environment.id,
-                    # pylint:disable=no-member
```

Comment:
I just thought these statements had any meaning for Host.create() which is actually treat parameters as properties and make_host() is a helper which is basically don't care what I pass there, but maybe I am wrong and need to return them. Should I?